stash_engine/app/models/stash_engine/curation_activity.rb

```diff
@@ -47,13 +46,6 @@ def submit_to_datacite
 
     private
 
-    def update_identifier_state
-      return if status == 'Status Unchanged'
-      return if stash_identifier.nil?
-      return if stash_identifier.identifier_state.nil?
-      stash_identifier.identifier_state.update_identifier_state(self)
-    end
-
     # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Removed updates to the `identifier_state` table and added association to `resource`